addons/l10n_in/data/account_tax_template_data.xml

```diff
@@ -219,8 +8,8 @@
         <field name="amount">5</field>
         <field name="tax_group_id" ref="cess_group"/>
         <field name="chart_template_id" ref="indian_chart_template_standard"/>
-        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),               (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11235'),                 'plus_report_expression_ids': [ref('tax_report_line_cess_tag')],             }),         ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             }),               (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p10055'),                 'minus_report_expression_ids': [ref('tax_report_line_cess_tag')],             }),         ]"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),             (0,0, {                 'factor_percent': 100,                 'repartition_type': 'base',             'tag_ids': [ref('tax_tag_base_cess')]}),               (0,0, {                 'factor_percent': 100,                 'repartition_type': 'tax',                 'account_id': ref('p11235'),                 'tag_ids': [ref('tax_tag_cess')],             }),         ]"/>
```

Comment:
Fix spacing in this file. I don't know if it was already like that, but for these, we want to use multiple lines, else it's unreadable.